src/client/main.cpp

```diff
@@ -76,9 +76,27 @@ int main(int argc, char* argv[]) {
 
   std::cout << "connecting to: " << arg_host << ":" << arg_port << std::endl;
 
-  //
-  // network code here
-  //
+  // NETWORK CODE
+  boost::asio::io_context io_context;
+  Addr server_addr{argv[1], argv[2]};
```

Comment:
intended for `arg_host` and `arg_port` to be used here

Reply:
Since we're replacing this with a config file, I think we can refactor these in a later PR.